backend/routes/userSession.js

```diff
@@ -0,0 +1,63 @@
+const express = require("express");
+const session = require('express-session');
+const {PrismaSessionStore} = require('@quixo3/prisma-session-store')
+const { PrismaClient } = require("@prisma/client");
+const prisma = new PrismaClient();
+const userSessionRouter = express.Router();
+
+const ONE_DAY = 86400000;
+const userSession = (session({
+    secret: process.env.SESSION_SECRET,
+    saveUninitialized: false,
+    resave: false,
+    cookie: {
+        secure: false,
+        httpOnly: true,
+        maxAge: ONE_DAY
+    },
+    store: new PrismaSessionStore(
+        prisma,
+        {
+            checkPeriod: 2 * 60 * 1000,  //ms
+            dbRecordIdIsSessionId: true,
+            dbRecordIdFunction: undefined,
+        },
+    )
+}));
```

Comment:
this should be creating your sessions for you already. What are you trying to use createSession for?